pkg/process/monitor/process_monitor.go

```diff
@@ -293,9 +286,8 @@ func (pm *ProcessMonitor) Initialize() error {
 					return
 				}
 				log.Errorf("process monitor error: %s", err)
-				pm.terminateProcessMonitor()
-				terminated = true
-				pm.Stop()
+				// closing netlink subscription
+				close(pm.done)
```

Comment:
you must use mutex here + you must set pm.done to nil after closing it, otherwise you'll panic as we're closing a closed channel

Reply:
thanks